scripts/web_aggregate_can_data.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
```

Comment:
This is the websockets thing I was talking about. Again, I don't think we should merge this script into master. If we need to refer to it we can always look at telemetry_xiv to see how we can recieve and decode CAN messages